lib/bindings_.ml

```diff
@@ -99,11 +99,11 @@ let send_case_req : type b t t_. {P:PS} -> (t, t_) reifyt -> t code -> (t_ -> P.
       (reifyk unrolledk, false)
 
 
-(* if the continuations are both static and equal then we can replace
+(* if the continuations are both static_ and equal then we can replace
```

Comment:
I'm not ecstatic_ about this change; it looks like the result of overeager search-and-replace.

Reply:
Oops, you're right. I'll give a thorough scan through and fix any others.